measures/ResStockArguments/measure.rb

```diff
@@ -730,37 +730,29 @@ def run(model, runner, user_arguments)
     if (args[:ducts_supply_location] != Constants.Auto) && (args[:ducts_supply_surface_area] == Constants.Auto)
       # set surface area based on OS-HPXML defaults
       f_out = (args[:geometry_unit_num_floors_above_grade] <= 1) ? 1.0 : 0.75
-      cfa_served = args[:geometry_unit_cfa] * [
-        args[:heating_system_fraction_heat_load_served] if args[:heating_system_type]=='Furnace', 
-        args[:cooling_system_fraction_cool_load_served] if args[:cooling_system_type]=='central air conditioner',
-        args[:heat_pump_fraction_heat_load_served] if args[:heat_pump_type]=='air-to-air',
-        args[:heat_pump_fraction_cool_load_served] if args[:heat_pump_type]=='air-to-air',
-      ].max
+      cfa_served = args[:geometry_unit_cfa] * get_max_heat_cool_load_served(args)
       primary_duct_area = 0.27 * cfa_served * f_out
       secondary_duct_area = 0.27 * cfa_served * (1.0 - f_out)
       if args[:ducts_supply_location] == 'Living Space'
```

Comment:
options_lookup.tsv has, e.g., `ducts_supply_location=living space` (note the case) so this might not work. Even better is to use an [HPXML constant](https://github.com/NREL/resstock/blob/da0d231421a492b9b7f28eaf1f24fab09395146e/resources/hpxml-measures/HPXMLtoOpenStudio/resources/hpxml.rb#L65-L402) so that the strings always stay in sync. So this would become `HPXML::LocationLivingSpace`.